metal/resource_metal_connection.go

```diff
@@ -0,0 +1,179 @@
+package metal
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/packethost/packngo"
+)
+
+func resourceMetalConnection() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceMetalConnectionRead,
+		Create: resourceMetalConnectionCreate,
+		Delete: resourceMetalConnectionDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the connection resource",
+				// remove ForceNew and do Update (needs packngo support)
```

Comment:
let's add `TODO:`
Do we have an issue open for this?

Reply:
https://github.com/packethost/packngo/issues/270